assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -21,109 +21,140 @@
-				background-size: 20px 19px;
-				border-color: $c-icon-notice-info;
-				border-radius: 0;
-				border-width: 0;
-				height: 19px;
-				width: 20px;
-			}
+			.mdc-select:not(.mdc-select--disabled) {
+
+				.mdc-floating-label {
+					color: $c-text-notice-info-selected-text;
+				}
+
+				.mdc-select__selected-text {
+					color: $c-text-notice-info-selected-text;
+				}
+
+				.mdc-select__dropdown-icon {
+					background-image: url("data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%231a73e8%22%20fill-rule%3D%22evenodd%22%20opacity%3D%221%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E");
+				}
 
-			> div + div {
-				margin-top: 32px;
+				.mdc-notched-outline__leading,
+				.mdc-notched-outline__notch,
+				.mdc-notched-outline__trailing {
+					border-color: $c-border-brand;
+				}
+
+				&:not(.mdc-select--focused) {
+
+					.mdc-select__selected-text:hover ~ .mdc-notched-outline {
+
+						.mdc-notched-outline__leading,
+						.mdc-notched-outline__notch,
+						.mdc-notched-outline__trailing {
+							border-color: $c-border-notice-info-hover;
+						}
+					}
+				}
 			}
 		}
 
-		.mdc-select:not(.mdc-select--disabled) {
+		.googlesitekit-settings-notice__learn-more {
+			flex-shrink: 0;
 
-			.mdc-floating-label {
-				color: $c-text-notice-info-selected-text;
+			@media (min-width: $bp-desktop) {
+				margin-left: $grid-gap-desktop;
 			}
+		}
 
-			.mdc-select__selected-text {
-				color: $c-text-notice-info-selected-text;
-			}
+		.googlesitekit-settings-notice__learn-more--align-bottom {
+			text-align: right;
 
-			.mdc-select__dropdown-icon {
-				background-image: url("data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%231a73e8%22%20fill-rule%3D%22evenodd%22%20opacity%3D%221%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E");
+			@media (min-width: $bp-tablet) {
+				display: flex;
+				flex-direction: column;
+				justify-content: flex-end;
+				padding-bottom: 4px;
+				padding-right: 12px;
+				text-align: left;
 			}
+		}
+
+		.googlesitekit-settings-notice__row--single-row {
```

Comment:
This styling might be getting too complex now :thinking: 

Instead I should probably have totally different styles for the single and multirow versions :thinking: 

It looks like it works on all viewport sizes though if you want to check

Can I set up VRT for this? I'm not sure what the criteria is to make something worthy of VRT.

Reply:
> Can I set up VRT for this? I'm not sure what the criteria is to make something worthy of VRT (I've not added a VRT yet)

Me neither :smile: Please, log a new ticket if you think it is worth a new VRT. Unfortunately, we don't have too much time for this because we are having the bug bash session for ga4 tomorrow and this PR needs to be merged asap.

Reply:
Sure thing @eugene-manuilov . As soon as we hear from Asvin let's merge it and I'll get the other merged